conanfile.py

```diff
@@ -28,7 +28,8 @@ class Log4cplusConan(ConanFile):
                "working_c_locale": [True, False],
                "decorated_name": [True, False],
                "qt4_debug_appender": [True, False],
-               "qt5_debug_appender": [True, False]}
+               "qt5_debug_appender": [True, False],
+               "unicode": ['ON', 'OFF']}
```

Comment:
why not `[True, False]` ?

Reply:
[True, False] options didn't work for me during testing.
Generated lib couldn't be linked by the application required log4cplus compiled with -DUNICODE=OFF option. I was getting "unresolved external symbol" errors

Reply:
Is there a better way of adding CMake build parameters without affecting Conan recipe?

Reply:
No. or you just customize the cmake file directly, or by Conan options.

Reply:
I've retested with [True, False] options.
Seems like it was only my error. Will change possible values

Reply:
Branch was updated